templates/README.md

```diff
@@ -1,41 +1,52 @@
 # Commonhaus Templates
 
-We don't want project on-boarding to be a burden.
+We don't want project on-boarding to be a burden, but we have a few requirements to ensure transparency.
 
-The files in this folder are our recommendations for what CF projects should define.
+- `CODE_OF_CONDUCT.md`[^1]: Projects must define a Code of Conduct. They can use the foundation's [Code of Conduct policy][coc_policy], or define their own. See [CODE_OF_CONDUCT.md][COC] and the [Project CODE_OF_CONDUCT.md template][COC_TPL] for examples.
```

Comment:
panda check allows .md,.adoc,.rst,.txt ...should i make it only greenlight .md ?

Reply:
I'm ok with addition that other txt-based formats are ok, too. Has to be obvious name, has to be plain text. Markup grammar is negotiable.